### PR TITLE
[DOC-3315] Update revoke tokens info

### PR DESCRIPTION
**Pull request by brian-f-harness**
Update revoke tokens info

For reviewers, [a preview](https://649b4c0ca48ff108aa1ce75e--harness-developer.netlify.app/docs/platform/Delegates/secure-delegates/secure-delegates-with-tokens#option-revoke-tokens) is available.

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [x] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [x] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

File: docs/platform/2_Delegates/secure-delegates/secure-delegates-with-tokens.md [hunk 1]
```diff
@@ -129,6 +129,10 @@ To update and restart an existing Docker delegate, do the following:
 
 ## Option: Revoke tokens
 
+Harness loads tokens during the delegate startup process as part of the connection heartbeat. When you change the delegate token, you must restart the delegate cycle process.
+
+Delegates are disconnected within 20 minutes of token revocation.
```
Comment (xingchi-jin) [inline docs/platform/2_Delegates/secure-delegates/secure-delegates-with-tokens.md:134]: i remember the self-destruct happens immediately when the token revokes. @jennyjames21  can you confirm and provide an estimation of time. I feel it doesn't take 20 min

Comment (jennyjames21) [reply docs/platform/2_Delegates/secure-delegates/secure-delegates-with-tokens.md:134]: yes it should happen immediately with latest change(PL-38957) which is not in prod yet

Comment (brian-f-harness) [reply docs/platform/2_Delegates/secure-delegates/secure-delegates-with-tokens.md:134]: Thank you @jennyjames21. @xingchi-jin for now, do we want to add the updated content in this PR and then update again when PL-38957 goes in to prod?

Comment (xingchi-jin) [reply docs/platform/2_Delegates/secure-delegates/secure-delegates-with-tokens.md:134]: Discussed from @brian-f-harness  in DM, the 20 min is actually an estimation but a relatively safe one . after a token is revoked, the next call delegate sends to acquire a task will disconnect the delegate. Altho we don't know when will the next acquire call be made by delegate
I think the best is, when Jenny's change is in prod, let's fix the doc to say immediately the delegate will be disconnected . For now, I will LG

**Review by xingchi-jin — APPROVED** [review]

File: docs/platform/2_Delegates/secure-delegates/secure-delegates-with-tokens.md (hunk 1 above)
Comment (brian-f-harness) [reply docs/platform/2_Delegates/secure-delegates/secure-delegates-with-tokens.md]: Updated content and will merge when PL-38957 goes in to prod.

Comment (xingchi-jin) [reply docs/platform/2_Delegates/secure-delegates/secure-delegates-with-tokens.md]: add a Jira https://harness.atlassian.net/browse/DOC-3329

**Review by douglas-j-bothwell — APPROVED** [review]

**Pull request merged**